other/quiz/quiz_questions.js

```diff
@@ -197,7 +197,11 @@ const pokemonType = () => new Promise(resolve => {
   (async () => {
     const pokemon = getRandomPokemon();
     const types = pokemon.type.map(t => PokemonType[t]);
-    const answer = new RegExp(`^\\W*(${types.join('\\W*')}|${types.reverse().join('\\W*')})\\b`, 'i');
+    let memeAnswer = null;
+    if (types.includes('Flying') && types.includes('Normal')) {
+      memeAnswer = 'bir[bd]';
+    }
+    const answer = new RegExp(`^\\W*(${types.join('\\W*')}|${types.reverse().join('\\W*')}${memeAnswer && `|${memeAnswer}`})\\b`, 'i');
```

Comment:
I think this outputs `null` as an answer when `memeAnswer` is null

Reply:
oh damn you are right my bad

Reply:
commited your suggestion